build-system/tasks/e2e/index.js

```diff
@@ -93,6 +93,7 @@ async function e2e() {
 
     // specify tests to run
     if (argv.files) {
+      // TODO: Should this allow multiple globs?
```

Comment:
Yeah, I'm sure this would be nice for local dev. I don't think the answer should change if watch is on. For simplicity's sake, always yes.

Reply:
Done.